File: src/main/java/com/wanderers/hotelier_webservices/server/service/api/CustomerService.java
package com.wanderers.hotelier_webservices.server.service.api;

import com.wanderers.hotelier_webservices.server.exception.CustomerServiceException;

public interface CustomerService {

    /**
     * Check if the customer exist in the datasource
     *
     * @param customerId
     * @return
     * @throws CustomerServiceException
     */
    Boolean isExistingCustomer(int customerId) throws CustomerServiceException;
}

File: src/main/kotlin/com/wanderers/hotelier_webservices/server/service/api/CustomerService.kt
package com.wanderers.hotelier_webservices.server.service.api

import com.wanderers.hotelier_webservices.server.exception.CustomerServiceException

interface CustomerService {

    /**
     * Check if the customer exist in the datasource
     *
     * @param customerId
     * @return
     * @throws CustomerServiceException
     */
    @kotlin.Throws(CustomerServiceException::class)
    fun isExistingCustomer(customerId: Int): Boolean
}